lib/pxclient.js

```diff
@@ -182,5 +209,25 @@ class PxClient {
             cb();
         }
     }
+
+    sendRemoteLog(message, severity, errorType, config) {
+        const enforcerConfig = new PxConfig(config, new PxLogger(config));
```

Comment:
This is confusing `new PxConfig(config, new PxLogger(config))`
Why not adding the logger in BDExpress instead?
If this is to avoid NPE inside the callServer then the validation the logger exists in the config instance should be inside callServer so it will secure all usages, not just the sendRemoteLog.
But you don't have to do it because this is not in the scope.

Reply:
I saw this but prefer to leave it as it is at this moment 👍

Reply:
Then lets do:
```
enforcerConfig.logger = enforcerConfig.logger ?  new PxLogger(config) : enforcerConfig.logger;
```